org.osate.ge.doc/docs/user_guide.md

```diff
@@ -281,9 +284,11 @@ When an element has been added to the diagram and needs to be positioned or size
 
 Table: Incremental Layout Modes {#tbl:incremental_layout_modes}
 
-### Known Layout Issues and Limitations
+#### Layout Feature Based on Direction
+When the *Layout Features Based on Direction* preference is enabled, the graphical editor will position ports based on the feature's direction. Input features will be placed on the left side and output features will be placed on the right side. If the preference is disabled, the layout algorithm will decide the side on which to place the feature.
 
-* Connections between multiple levels of the diagram hierarchy are not routed out. Such connections usually represent property values such as bindings. Such connections must be routed manually.
+### Known Layout Issues and Limitations
+* Connections between multiple levels of the diagram hierarchy are not routed. Such connections usually represent property values such as bindings. Such connections must be routed manually.
```

Comment:
"Such connections usually represent property values such as bindings. Such connections must be routed manually."

I think combining these sentences would be better. Just a thought.

Possible changes:

    "Such connections must be routed manually and usually represent property values such as bindings."
or

    "Such connections usually represent property values such as bindings and must be routed manually."